pages/admin/invite/userRequest.tsx

```diff
@@ -0,0 +1,91 @@
+import { useEffect, useState } from "react";
```

Comment:
userRequest should be a component, not a page. I've said this multiple times so please turn it into a component